src/NZGBplugin/LINZ/gazetteer/gui/NameSearchWidget.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import print_function
```

Comment:
Style only changes ?

Reply:
again adding in missing PyQt5.QtWidgets import